BraintreeDropIn/Public/BTDropInRequest.h

```diff
@@ -41,6 +41,11 @@ typedef NS_ENUM(NSInteger, BTFormFieldSetting) {
 /// Defaults to false.
 @property (nonatomic, assign) BOOL threeDSecureVerification;
 
+/// Optional: Enable 3DS verification and specify options and additional information. If no amount is set, the `BTDropInRequest` `amount` will be used.
+///
+/// Note: For 3DS 2 options, set `billingAddress`, `amount`, `email`, `mobilePhone` for best results.
```

Comment:
Instead of "for 3DS 2 options", can we clarify this comment? Maybe something along the lines of "To encourage 3DS 2.0 flows".